tests/requirements.txt

```diff
@@ -1 +1,2 @@
-pytest
\ No newline at end of file
+pytest
+pyonenote
```

Comment:
pyonenote is a service, and not test dependency so it should be put in a new top level requirements.txt and not the test/requirements.txt